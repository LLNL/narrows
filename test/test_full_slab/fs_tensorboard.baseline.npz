version https://git-lfs.github.com/spec/v1
oid sha256:f1a0a601802f0bc6b24cd8103e3179580f63e7babd7605d07d12c20942970b20
size 9290
