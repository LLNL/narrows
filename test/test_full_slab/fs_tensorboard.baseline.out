Executable       : N/A
Command line args: /home/mike/Documents/work/narrows/repo/clean/test/test_full_slab/../full_slab.yaml -d out=fs_tensorboard -d epsilon=0.1 -d tensorboard=True
Narrows version  : 0.1-9-5d6c739
Computer name    : gannon
Run date         : 07 August 2020 (Friday)
Run time         : 18:11:26 -0700

Copyright (c) 2020
Lawrence Livermore National Security, LLC
All Rights Reserved

Python version   : 3.8.2
flake8           : 3.8.1
matplotlib       : 3.2.1
numpy            : 1.18.4
pandas           : 1.0.3
pytest           : 5.4.2
pyyaml           : 5.3.1
scipy            : 1.4.1
torch            : 1.5.0

bc:
  bc1:
    behavior: vacuum
    location: 0.0
  bc2:
    behavior: vacuum
    location: 1.0
ctrl:
  cells_per_region: 50
  epsilon: 0.1
  hidden_layer_nodes: 5
  learning_rate: 0.001
  mc: true
  num_particles: 1000000.0
  num_physical_particles: 8
  ordinates: 4
  out: fs_tensorboard
  sn: true
  sn_epsilon: 1.0e-06
  tensorboard: true
mat:
  mat1:
    sigma_a: 8.0
    sigma_s0: 0.0
    sigma_s1: 0.0
reg:
  reg1:
    end: 1.0
    mat: mat1
    start: 0.0
src:
  src1:
    end: 1.0
    magnitude: 8.0
    start: 0.0

Training neural network
Iter 0: 2898.8154296875
Iter 500: 76.31524658203125
Iter 725: 38.34339141845703
Running MC
Running SN
         Runtime
Train   1.66e+00
Predict 2.68e-04
MC      1.53e+00
SN      2.39e-02

Narrows is finished.
